vendor.conf

```diff
@@ -123,7 +123,7 @@ github.com/googleapis/gax-go                        317e0006254c44a0ac427cc52a0e
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    c80284d4b5291a351bb471bcdabb5c1d95e7a583 # master / v1.4.0-dev
+github.com/containerd/containerd                    5c695da77b0823df3b4a79100f0fb63f9ce6446b # master / v1.4.0-beta.1-73-g5c695da7
```

Comment:
the dependencies of containerd should be renvendored as well

Reply:
Thanks. Let me modify it.

Reply:
Addressed.